experimental/dds/tree2/src/feature-libraries/editable-tree-2/editableTreeTypes.ts

```diff
@@ -125,6 +125,11 @@ export interface TreeNode extends Tree<TreeSchema> {
 		listener: EditableTreeEvents[K],
 	): () => void;
 
+	/**
+	 * Gets a field of this node..
+	 */
+	getField(key: FieldKey): undefined | TreeField;
```

Comment:
The implementation of this method never returns undefined. Why is it typed such that it can?